Sources/MySQLKit/MySQLDatabase.swift

```diff
@@ -201,4 +203,9 @@ public struct MySQLDialect: SQLDialect {
             alterColumnDefinitionTypeKeyword: nil
         )
     }
+    
+    public func normalizeSQLConstraint(identifier: SQLExpression) -> SQLExpression {
+        guard let sqlIdentifier = identifier as? SQLIdentifier else { return identifier }
```

Comment:
How I imagined this was having something like `SQLHashedExpression` that serializes the expression by hashing it. So this method would be:

```swift
return SQLHashedExpression(identifier)
```

`SQLHashedExpression` would be implemented like:

```swift
func serialize(to serializer: SQLSerializer) {
    self.expression.serialize(to: SQLHashingSerializer(serializer))
}
```

Lmk if that makes sense.